Makefile

```diff
@@ -57,20 +57,24 @@ ghci:
 
 # TEST
 
-test: testunix testexamples
+test: testunix testexamples testbash
 
 # run cross-platform and unix-specific shell tests with default shelltest build
 testunix: build
-	$(SHELLTEST) tests -x /examples -x .windows -w $(DEFAULTEXE)
+	$(SHELLTEST) tests -x /bash -x /examples -x .windows -w $(DEFAULTEXE)
 
 # run cross-platform and windows-specific shell tests
 # (though if you are running make on windows, you may be able to, or have to, use testunix)
 testwindows:
-	$(SHELLTEST) tests -x /examples -x .unix -w $(DEFAULTEXE)
+	$(SHELLTEST) tests -x /bash -x /examples -x .unix -w $(DEFAULTEXE)
 
 testexamples: build
 	$(SHELLTEST) tests/examples
 
+testbash: build
+	! $(SHELLTEST) tests/bash
```

Comment:
@obfusk, what is this syntax ? (!)

Although `make testbash` passes its test, make then exits with an error, which we don't want.

What's the purpose of tests/bash again ? This is a place to put tests which test bash-specific things, and in theory should be run only with bash I guess.

What is dollar-quote.test for.. is it #15 ? Perhaps worth adding the issue number and a one line description of what's being tested, to the test file.

Reply:
From `man bash`: "If the reserved word !  precedes a  pipeline, the exit status of that pipeline is the logical negation of the exit status".
The `dollar-quote.test` is indeed for #15. I'm running it both without bash (expecting it to fail) and with bash (expecting it to succeed).
The test that's expected to fail indeed shows an error in the output, but because its exit status is negated, the `make test` command as a whole succeeds.